verde/utils.py

```diff
@@ -201,3 +203,49 @@ def maxabs(*args):
     arrays = [np.atleast_1d(i) for i in args]
     absolute = [np.abs([i.min(), i.max()]).max() for i in arrays]
     return np.max(absolute)
+
+
+def grid_to_table(grid):
+    """
+    Converts an xarray Dataset grid to a Pandas DataFrame
+
+    Parameters
+    ----------
+    grid
```

Comment:
```suggestion
    grid : :class:`xarray.Dataset`
```

This way sphinx will create a nice link to the xarray docs.